### PR TITLE
BarGuage: updates story from knobs to controls

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR updates story from knobs to controls
**Which issue(s) this PR fixes**:
Relates to #30182

### CODE REVIEW

File: packages/grafana-ui/src/components/BarGauge/BarGauge.story.tsx [hunk 1]
```diff
@@ -84,23 +63,54 @@ function addBarGaugeStory(overrides: Partial<Props>) {
...
+lcdHorizontal.args = {
+  value: 70,
+  title: 'Title',
+  minValue: 0,
+  maxValue: 100,
+  threshold1Value: 40,
+  threshold1Color: 'orange',
+  threshold2Value: 60,
+  threshold2Color: 'red',
+};
```
Comment (torkelo) [inline packages/grafana-ui/src/components/BarGauge/BarGauge.story.tsx:116]: These seems to be all set to the same value, you can just do 
gradientVertical.args = gradientHorizontal.args = lcdHorizontal.args

**Review by peterholmberg — CHANGES_REQUESTED** [review]
If you look at what Props the BarGauge has, there's no support to change any of them. We however have a lot of controls to change other things that are not directly tied to the component. I feel we should focus more on the look and feel of the actual component props rather than underlying things (like value and thresholds which is a nice plus I suppose).

**Comment by thisisobate** [comment]
True...I agree with you @peterholmberg
I just added support for some useful props

File: packages/grafana-ui/src/components/BarGauge/BarGauge.story.tsx [hunk 2]
```diff
@@ -28,35 +15,110 @@ export default {
...
+    showUnfilled: {
+      table: {
+        disable: true,
+      },
+    },
+    alignmentFactors: {
+      table: {
+        disable: true,
+      },
+    },
```
[the remark this answers is not in the document]
Comment (thisisobate) [reply packages/grafana-ui/src/components/BarGauge/BarGauge.story.tsx:91]: So...these args are coming from the BarGauge Props.
Storybook automatically generates controls based on React components.

**Review by torkelo — APPROVED** [review]

**Review by peterholmberg — APPROVED** [review]
Very nice :ok_hand:

**Comment by hugohaggmark** [comment]
Looking at this today and they're a lot of properties that aren't meaningful for the story like theme, field, value etc. Weren't we supposed to hide these? WDYT @torkelo?

image

**Comment by torkelo** [comment]
Yes I thought so

**Comment by hugohaggmark** [comment]
I've updated the issue with my review of all migrated controls so far.

**Pull request merged**